backend/ModelLayer/Helper/MailValidator.cs

```diff
@@ -10,6 +10,10 @@ public class MailValidator : ValidationAttribute
     {
         public override bool IsValid(object value)
         {
+            if (value is string text && text.Length == 0)
```

Comment:
passt das mit text.length == 0

der in der Bedingung gibst du ja ein True zurück.

kommt mir komisch vor

Reply:
das ist beabsichtigt so um keine notwendigkeit dieses feldes zu erzeugen..bedeutet entweder man lässt es vorläufig leer oder muss dann halt schauen das des format passt..denke das ist sinnvoll so..ansonsten haben wir zuviele pflichtfelder

Reply:
aber wäre es dann nicht möglich dieses Feld leer zu lassen?

Reply:
ne weil wenn dieses statement nicht da wäre, er in jedem fall irgendeine eingabe erwarten würde..deshalb bin ich ja überhaupt darauf aufmerksam geworden